### PR TITLE
Add sector to filter options and list display for omis orders

### PR DESCRIPTION
**Pull request by ztolley**

### CODE REVIEW

**Comment by reupen** [comment]
The filter doesn't seem to be having any effect. Is it using `sector_descends`?

File: src/apps/omis/apps/list/controllers.js [hunk 1]
```diff
@@ -1,31 +1,81 @@
...
+      selectedFilters,
+      filtersFields,
+      countLabel: 'order',
+      actionButtons: [{
+        label: 'Add order',
+        url: '/omis/create',
+      }],
+    })
+  } catch (error) {
+    console.log(error)
```
Comment (reupen) [inline src/apps/omis/apps/list/controllers.js]: Did you mean to put a `console.log()` here?

**Comment by reupen** [comment]
Now it's having an effect, but if you select and change the sectors too quickly, you end up with the wrong results.

**Comment by ztolley** [comment]
The issue with clicking filters too fast it a general problem on all filter pages, caused by how the XHR module works. Maybe time to revisit it as part of moving the code to a shared framework

**Pull request merged**